EventFilter/HGCalRawToDigi/test/HGCalUnpackerTest.cc

```diff
@@ -15,50 +15,52 @@ uint16_t enabledERXMapping(uint16_t sLink, uint8_t captureBlock, uint8_t econd)
 HGCalElectronicsId logicalMapping(HGCalElectronicsId elecID) { return elecID; }
 
 int main(int argc, char* argv[]) {
```

Comment:
Not very important as this is just a test, but what is the purpose of keeping an empty test ?

Reply:
we have removed this test now - it served it's initial purpose (before we had full system test data)